packages/shared/components/inputs/NetworkInput.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="typescript">
+    import { DestinationNetwork } from '@core/network'
+    import { Modal, NetworkSelector, SelectorInput } from 'shared/components'
+
+    export let network: DestinationNetwork = DestinationNetwork.Shimmer
+    export let error: string = ''
+
+    let inputElement: HTMLInputElement = undefined
+    let modal: Modal = undefined
+</script>
+
+<SelectorInput
+    labelLocale="general.destinationNetwork"
+    bind:value={network}
+    bind:inputElement
+    bind:modal
+    {error}
+    readonly
```

Comment:
could you update the pointer event if its `readonly`

Reply:
I added it to the selector input. It's not perfect, but a proper fix would require extracting the button type from the `InputContainer` and that should be part of a wider refactor.